package.json

```
@@ -1,8 +1,22 @@
 {
   "name": "use-polling",
   "version": "0.1.3",
-  "main": "index.js",
+  "main": "lib/index.js",
   "author": "YangKui <c6swift@gmail.com>",
   "license": "MIT",
-  "dependencies": {}
+  "scripts": {
+    "build:babel": "npx babel index.js --out-file lib/index.js",
+    "build:umd": "npx webpack --progress --mode=production"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/preset-env": "^7.1.6",
+    "babel-loader": "^8.0.4",
+    "webpack": "^4.26.0",
+    "webpack-cli": "^3.1.2"
+  },
+  "dependencies": {
+    "@babel/polyfill": "^7.0.0"
```

```suggestion
    "@babel/polyfill": "^7.0.0"
```
为什么这里是 `dependencies`

---

依赖 babel-polyfill 的，用 lib/ 下的是直接引用，dist/ 下的是把 polyfill 打到一个文件里